### DISCUSSION TITLE
饼图设置size通道时，能否从中间往外渲染，如下图效果

### DISCUSSION THREAD

**Discussion by Can-Xu** [antvis/G2#5453] [Q&A]
![image](https://github.com/antvis/G2/assets/37643989/399da6ef-e6a3-4bf0-8c57-d4cb39fa29f5)

**Comment by hustcc** [comment]
没有理解这个意思是什么？

**Reply by Can-Xu** [reply]
![image](https://github.com/antvis/G2/assets/37643989/b2909c5b-298a-437a-919b-f0e419616154)
![image](https://github.com/antvis/G2/assets/37643989/7998c3b8-3d58-4a57-af42-81de5a468691)

**Reply by hustcc** [reply]
应该是需要参考这个 demo：https://g2.antv.antgroup.com/examples/general/rose/#rose-label
